lib/ecl/ecl_smspec.cpp

```diff
@@ -1648,13 +1648,10 @@ const char * ecl_smspec_get_restart_case( const ecl_smspec_type * ecl_smspec) {
     return NULL;
 }
 
-
-const float * ecl_smspec_get_params_default( const ecl_smspec_type * ecl_smspec ) {
-  return ecl_smspec->params_default.data();
+const std::vector<float>& ecl_smspec_get_params_default( const ecl_smspec_type * ecl_smspec ) {
```

Comment:
If you implement this as:
```C++
const float * ecl_smspec_get_params_default( ) {
     return ecl_smspec->params_default.data();
}
```

you can get away without API changes.

Reply:
True, and this is the solution used in master. But, if a std::vector is returned, use of ecl_smspec_get_params_default is simpler and one does not have to resort to raw pointer operations.